posts/201804.md

```diff
@@ -4,40 +4,53 @@
 
-The release process has seen an introduction of Release Candidates. It used to be: announce release date, rush to meet it and miss bugs. Now it is: don't call release date, post Release Candidate 1 (RC1) and let people start testing it, fix bugs and release RC2, repeat. An RC version with no apparent bugs becomes the final release. This new process removes a ton of pressure from developers and users and gives more time for testing. As our primary consumer-facing product, Decrediton, is growing in features and complexity, more testing will be required for new releases.
+The release process has been improved. Instead of announcing a release date and trying to meet it, a Release Candidate 1 (RC1) will now be posted. After it has been tested an bugfixed with the help of the community, a second candidate (RC2) will be released. This is repeated until an RC version with no apparent bugs becomes the final release. The new process removes a ton of pressure from developers and users and gives more time for testing. As our primary consumer-facing product, Decrediton, is growing in features and complexity, more testing will be required for new releases.
+#reasoning:
+#flow & avoids internal conflict by preceding information:
+#A) for Decredition: use bug fix 1.2.1 release
+#B) -> we have a new procedure: 'used to be: ... miss bugs' (obviously we're still missing bugs)
+#
+#if you don't like the suggestivity of 'improved' you can say 'changed'
 
 Politeia is "Getting close to a public beta of voting" ([slack](https://decred.slack.com/archives/C0NRMDVMJ/p1523623105000168)). Decred plugin [merged](https://github.com/decred/politeia/pull/242), paywall and voting are in the testing stage. Ticket voting [works](https://twitter.com/decredproject/status/984518284757221376) on testnet via CLI.
 
+Decred.org received a new sleek [exchanges page](https://www.decred.org/exchanges/).
+
 Trezor support advanced with Decred PR [merged](https://github.com/trezor/trezor-mcu/pull/274). Please note this is only firmware support, to be usable it also needs wallet integration.
 
 WooCommerce [Decred plugin](https://github.com/xifrat/decred-woocommerce-plugin) alpha version is ready for testing.
-
-Website received a new sleek [exchanges](https://decred.org/exchanges/) page and many additions to a list of [contributors](https://decred.org/contributors/).
+ 
+This month, the project has grow by x [contributors](https://www.decred.org/contributors/).
+#comment: would be nice if we could find out a number for that. If its not possible, I would rephrase to: 
+#Plenty of new [contributors](https://www.decred.org/contributors/) have joined the project this month.
+#I strongly prefer this over stating that the contributers list on the website has been updated, because that's too much stating #the obvious imo
```

Comment:
Well, "many additions to" implies growth, but in your version I like that it's more explicit. I was wanting this number in #2 . Maybe I could derive it from dcrweb updates, I'll have a look. Oh and "grow**n**".

One thing I'm not sure is whether to equate the list of contributors at decred.org to "the project".